models/agent_work_time/int_zendesk__ticket_work_time_calendar.sql

```diff
@@ -12,6 +12,8 @@ with ticket_historical_status as (
             else 0 end as agent_wait_time_in_minutes,
         case when status in ('new', 'open', 'hold') then status_duration_calendar_minutes
             else 0 end as requester_wait_time_in_minutes,
+        case when status in ('new', 'open', 'hold', 'pending') then status_duration_calendar_minutes 
+            else 0 end as solve_time_in_minutes, 
```

Comment:
can we go over why this is defined twice in two separate places? Is there an opportunity for us to follow DRY principles?

Reply:
Hi @fivetran-jessicacherny ! These metrics are being constructed for two different models here--with schedules enabled in Zendesk, and without schedules. 

- The `int_zendesk__ticket_work_time_calendar.solve_time_in_minutes` field is when schedules are disabled (as I believe they are internally). This is the field being brought downstream into our internal models.
- The `int_zendesk__ticket_work_time_business.solve_time_in_minutes` field is when schedules are enabled (as they are for many other customers). This field is not currently being brought downstream internally.

Let me know if that clears it up!

Reply:
definitely does! thank you @fivetran-avinash!